run/test/specs/message_community_invitation.spec.ts

```diff
@@ -1,17 +1,19 @@
 import { androidIt, iosIt } from '../../types/sessionIt';
+import { InviteContactsButton } from './locators';
 import { sleepFor } from './utils';
 import { newUser } from './utils/create_account';
 import { newContact } from './utils/create_contact';
 import { joinCommunity } from './utils/join_community';
-import { SupportedPlatformsType, openAppTwoDevices, closeApp } from './utils/open_app';
+import { SupportedPlatformsType, closeApp, openAppTwoDevices } from './utils/open_app';
 
 iosIt('Send community invitation', sendCommunityInvitationIos);
 androidIt('Send community invitation', sendCommunityInviteMessageAndroid);
 
+const communityLink = `https://chat.lokinet.dev/testing-all-the-things?public_key=1d7e7f92b1ed3643855c98ecac02fc7274033a3467653f047d6e433540c03f17`;
+const communityName = 'Testing All The Things!';
```

Comment:
you should create a file with those exported and import them everywhere, instead of duplicating them a bunch of times

Reply:
done

Reply:
So I see you did this in `run/constants/community.ts` but you didn't remove this code and then import `testCommunityLink` and `testcommunityName` here and in `disappearing_community_invite.spec.ts`?

Reply:
good catch, have updated